modules/php/Models/Astra.php

```diff
@@ -185,23 +183,31 @@ public function isEndOfGameTriggered(): bool
     return false;
   }
 
-
   /**
-   * UI DATA
+   * @return int
    */
-  public function getUiData()
+  public function getCardsByActionMap(): array
   {
-    $data = [];
-    $totalScore = 0;
-
-    // Count number of cards of each action
```

Comment:
That's not what happened, Github! I just moved a part code from getUiData() to a separate function!